cr8/clients.py

```diff
@@ -172,6 +173,34 @@ def _to_dsn(hosts):
     return dsn
 
 
+def _to_boolean(v):
+    if str(v).lower() in ("yes", "true", "t", "1"):
```

Comment:
I'd make this stricter and just support `true` and `false`, or did you do it that way for compatibility with something else?